src/templates/author-template.tsx

```diff
@@ -24,25 +24,23 @@ const AuthorPageTemplate: React.FC<AuthorTemplateProp> = ({ data }) => {
   const [name, setName] = React.useState(undefined)
 
   return (
-    <div>
-      <Layout>
-        <h1>
-          {React.useEffect(() => {
-            setName(localStorage.getItem("name"))
-          }, [])}
-          {name}:
-        </h1>
-        {allNodes.map((singleNode, index) => {
-          const { date, title } = singleNode.frontmatter
-          return (
-            <div key={index}>
-              <Link to={`/blogs/${_.kebabCase(title)}`}>{title}</Link>
-              <small>{date}</small>
-            </div>
-          )
-        })}
-      </Layout>
-    </div>
+    <Layout>
+      <h1>
+        {React.useEffect(() => {
+          setName(localStorage.getItem("name"))
+        }, [])}
+        {name}:
```

Comment:
Can this hook be moved into the main body of the function?

Reply:
> Can this hook be moved into the main body of the function?

Yes. Will do.